deploy/dockerephemeral/docker-compose.yaml

```diff
@@ -60,7 +60,8 @@ services:
   basic_smtp: # needed for demo setup
     container_name: demo_wire_smtp
 #    image: namshi/smtp
-    image: julialongtin/smtp:0.0.9
+#    image: julialongtin/smtp:0.0.9
```

Comment:
IMHO we may delete these two lines: If anyone cares about the history of images, they may have a look at the git log.